client/src/components/Cost.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const CostWrapper = styled.section`
+  float:left;
+  font-size: 22px;
+  font-family: Circular, -apple-system, BlinkMacSystemFont, Roboto, "Helvetica Neue", sans-serif;
+  font-weight: 800;
+  color: rgb(72, 72, 72);
+  overflow-wrap: break-word;
+  line-height: 1.44444em; 
+
+`;
+
+const NightWrapper = styled.section`
+  float: left;
+  font-size: 12px !important;
+  font-weight: 600 !important;
+  
```

Comment:
Empty line breaks before the closing back tick for both NightWrapper and CostWrapper